styles/main.css

```diff
@@ -174,8 +178,12 @@ div .paperclip {
 .back-to-top {
 	text-align: right;
 	margin-right: -0.5em;
-	opacity: 50%;
-	margin-bottom: 3.5em;

```

Comment:
This should be done using the 'bottom' property in conjunction with the hyperlink's fixed position